mern/client/src/screens/StartingIntent/StartingIntent.js

```diff
@@ -1,32 +1,45 @@
-import React from "react";
-import { useContext } from "react";
+import React, { useContext } from "react";
 import styles from "./StartingIntent.module.css";
 import "./../../Components/Buttons/ButtonStyleSheet.css";
 import GenericButton from "../../Components/Buttons/GenericButton";
 import IntentButtons from "../../Components/IntentButtons/IntentButtons";
-import { useNavigate } from "react-router-dom";
+// import { useNavigate } from "react-router-dom";
 import Scrollbar from "../../Components/TranscriptScroller/transcript-scroller.component";
-import { deleteFile } from "../../utils/transcript";
+import { recoverFlow } from "../../Controller/recoverSessionController"
+// import { deleteFile } from "../../utils/transcript";
 import { SessionContext } from "../../Contexts/sessionProvider";
 import { SpeakerContext } from "../../Contexts/speakerProvider";
 import { IntentContext } from "../../Contexts/intentsProvider";
 
 function StartingIntent() {
-  const [, , transcriptID] = useContext(SessionContext);
+  const [sessionID, , ] = useContext(SessionContext);
   const [currSpeaker, setSpeaker, prevSpeaker, setPrevSpeaker] =
     useContext(SpeakerContext);
   const [intentState] = useContext(IntentContext);
-  const Navigate = useNavigate();
-  const PageChange = (url) => {
-    Navigate(url);
-  };
+  // const Navigate = useNavigate();
+
+  // const PageChange = (url) => {
+  //   Navigate(url);
+  // };
```

Comment:
maybe you can delete this and the commented lines above if it is not useful anymore? totally up to you though! :))

Reply:
Deleted it! I thought we might need it in the future, but I'm now realizing the starting-intent page doesn't reroute to any other pages.